src/modules/fancyzones/lib/Settings.cpp

```diff
@@ -109,10 +115,15 @@ IFACEMETHODIMP_(void) FancyZonesSettings::CallCustomAction(PCWSTR action) noexce
 }
 CATCH_LOG();
 
-void FancyZonesSettings::LoadSettings(PCWSTR config, bool fromFile) noexcept try
+bool FancyZonesSettings::LoadSettings(PCWSTR config, bool fromFile) noexcept try
 {
+    if (!config)

```

Comment:
Why do we need this check now?

Reply:
Again, check is added because there are unit tests, testing this scenario. Not sure if it can happen in real scenario because the function that is providing this resource string has fallback scenario covered. Should I keep this and that unit test, or remove both of them ?